selfdrive/car/hyundai/values.py

```diff
@@ -32,7 +32,7 @@ class CAR:
   IONIQ_EV_2020 = "HYUNDAI IONIQ ELECTRIC 2020"
   IONIQ_PHEV = "HYUNDAI IONIQ PHEV 2020"
   KONA = "HYUNDAI KONA 2020"
-  KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
+  KONA_EV = "HYUNDAI KONA ELECTRIC 2018-2019"
```

Comment:
```suggestion
  KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
```
We try to not change car strings after the initial port.